libraries/state_history/abi.cpp

```diff
@@ -133,6 +133,22 @@ extern const char* const state_history_plugin_abi = R"({
                 { "name": "delta", "type": "int64" }
             ]
         },
+        {
+            "name": "sync_call_trace", "fields": [
```

Comment:
I guess this is where my comment https://github.com/AntelopeIO/abieos/pull/42/files#r2036377296 should have originally gone. ship does a good job versioning everything, so I think we ought to have this as `sync_call_trace_v0` to follow existing pattern.

I think it probably ought to be a variant as well, to match existing established pattern. Not sure how problematic that may be off hand.